### DISCUSSION TITLE
Hypothes.is templater script

### DISCUSSION THREAD

**Discussion by TfTHacker** [SilentVoid13/Templater#156] [Templates Showcase]
Posted a script for working with Hypothes.is

https://forum.obsidian.md/t/retrieve-annotations-for-hypothes-is-via-templater-plugin-hypothes-idian/17225

Templater is great! Thank you for good docs. Submitted some issues based on dev experience.

**Comment by SilentVoid13** [comment]
Thanks for sharing!
